src/entry.js

```diff
@@ -1,6 +1,20 @@
+import React from 'react'
 import ReactDOM from 'react-dom'
+import { Provider } from 'react-redux'
+
 import routes from './routes'
+import store from './store'
+
 import '../sass/app.scss'
 
-const div = document.getElementById('app')
-ReactDOM.render(routes, div)
+const rootEl = document.getElementById('app')
+const render = () => {
```

Comment:
Well, where the `<Provider />` component is, it needs access to the store since it is what passes it converts the state to an object and passes it down the tree.